frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/modal-footer.tsx

```diff
@@ -12,87 +13,170 @@ import {
   BlockPoolActionType,
   checkRequiredValues,
   BlockPoolState,
+  FooterPrimaryActions,
 } from '../../../utils/block-pool';
 
 export const BlockPoolModalFooter = (props: BlockPoolModalFooterProps) => {
-  const {
-    state,
-    dispatch,
-    onPoolCreation,
-    onClick,
-    actionLabel,
-    closeLabel,
-    cancel,
-    close,
-  } = props;
+  const { state, dispatch, onSubmit, footerPrimaryAction, cancel, close } = props;
```

Comment:
Can't this footer be used for create block pool as well ?
Just to have one footer - `BlockPoolFooter` rather than one for modals and one for create.

Reply:
I can but it needs some hack for alignment, l thought it will confuse others, anyway, let me think what I can do here